tests/common/plugins/conditional_mark/tests_mark_conditions_drop_packets.yaml

```diff
@@ -163,6 +163,27 @@ drop_packets/test_drop_counters.py::test_loopback_filter:
   skip:
     reason: "SONiC can't enable loop-back filter feature"
 
+drop_packets/test_drop_counters.py::test_multicast_smac_drop[port_channel_members]:
+  skip:
+    reason: "Test case requires explicit fanout support"
+    strict: True
+    conditions:
+      - "hwsku in ['Cisco-8101-O8C48']"
+
+drop_packets/test_drop_counters.py::test_multicast_smac_drop[vlan_members]:
+  skip:
+    reason: "Test case is only suitable for t0 type topology since it requires vlan interfaces"
+    strict: True
+    conditions:
+      - "hwsku in ['Cisco-8101-O8C48']"
+
+drop_packets/test_drop_counters.py::test_multicast_smac_drop[rif_members]:
+  skip:
+    reason: "No rif_members available"
+    strict: True
+    conditions:
+      - "hwsku in ['Cisco-8101-O8C48']"
```

Comment:
@XuChen-MSFT why only for Cisco SKUs?

Reply:
@kperumalbfn  these test cases are already expected to be skipped. However, on Cisco SKUs the tests still go through the setup phase before reaching the skip logic, and the setup stage hits some environment‑related issues. This leads to unnecessary setup execution and false failures that are not related to the actual test logic.
By moving the skip condition into the mark/collection phase, we skip these tests earlier, avoid triggering the setup routines, prevent environment‑related setup failures, and also speed up overall test execution